tests/test_integration/test_commands/test_migrate_prepare.py

```diff
@@ -16,7 +11,11 @@ def test_migration_prepare_command(
 
         dry_run_result = cmd.deploy_cognite_migration(toolkit_client, True, verbose=False)
 
-        actual_result = cmd.deploy_cognite_migration(toolkit_client, False, verbose=False)
+        actual_result = cmd.deploy_cognite_migration(
+            toolkit_client,
+            False,
+            verbose=False,
+        )
 
         assert dry_run_result.has_counts, "Dry run should have counts"
```

Comment:
this test only compares `dry_run` to `actual_result` (both are implementations)
for the test to be complete, you'd need to compare this to a static value that you expect.
also, `DeployResults` is a bigger object, why only one property is tested?

Reply:
Main point of this test is ensure end-to-end. The deployment logic is thoroughly tested elsewhere. However, you are right it can be improved.